packages/flutterfire_ui/lib/src/auth/oauth/providers/twitter_provider.dart

```diff
@@ -88,7 +91,7 @@ class TwitterProviderConfiguration extends OAuthProviderConfiguration {
   String get providerId => TWITTER_PROVIDER_ID;
 
   @override
-  OAuthProvider createProvider() {
+  TwitterProvider createProvider() {
```

Comment:
Are these not breaking changes?

Reply:
if anyone using `createProvider` directly (that I doubt anyone does), this shouldn't break anything unless I am missing something. Do you have a scenario in mind where this would break something?

Reply:
Just saw return type change and thought breaking change, but if these are internal use then let's ignore my comment :D